server/routes/shared/riskPresenter.ts

```diff
@@ -8,7 +9,9 @@ export interface RoshAnalysisTableRow {
 export default class RiskPresenter {
   constructor(private readonly riskSummary: RiskSummary | null) {}
 
-  readonly riskSummaryEnabled = this.riskSummary !== null
+  readonly riskSummaryEnabled = config.apis.assessRisksAndNeedsApi.riskSummaryEnabled
+
+  readonly riskSummaryNotFound = this.riskSummary === null
```

Comment:
this legitimizes `null` as a value for `riskSummary`. therefore i changed the enabled flag to be based on the config only.